README.md

```diff
@@ -20,17 +20,26 @@ See [Releasing a new version of the Sling starter](https://cwiki.apache.org/conf
         directory. It is advisable to use a work directory outside of the
         project directory.
 
-1) Build the Sling Starter using 
+1) Build the Sling Starter using `mvn clean install` in the current directory.
+2) Copy the launch to project directory `cp target/dependency/org.apache.sling.feature.launcher.jar $path-to-project`
+3) Copy either the feature archive (*.far) or individual set of feature.json files to the project folder 
 
-	mvn clean install
-	
-in the current directory.
 
-2) Start Sling backed by an Oak SegmentStore with
+Example 1: Start an instance using the launcher specifying a feature archive (.far)
+This example uses the aggregate `oak_tar_far` as defined in the pom.xml
 
-    java -jar target/dependency/org.apache.sling.feature.launcher.jar -f target/slingfeature-tmp/feature-oak_tar.json
-	
-3) Browse Sling in:
+        java -jar org.apache.sling.feature.launcher.jar -f org.apache.sling.starter-12-SNAPSHOT-oak_tar_far.far 
+	 
+Example 2: Start an instance using the launcher specifying an aggregate feature file. 
+This example uses the aggregate `feature-oak_tar.json` as defined in the pom.xml
+
+        java -jar org.apache.sling.feature.launcher.jar -f feature-oak_tar.json
+
+Example 3: Start an instance using the launcher specifying a set feature files.
+ 
+        java -jar org.apache.sling.feature.launcher.jar -f feature-base.json,feature-boot.json,....
```

Comment:
Perhaps it would be better to point to the feature model launcher documentation instead?